.github/workflows/ci.yaml

```diff
@@ -18,24 +18,25 @@ jobs:
         build_type: [release, debugoptimized]
 
     steps:
-    - name: Setup system
+    - uses: actions/checkout@
```

Comment:
This must be checkout@v2

Reply:
Looks like I dropped something along the way...